core/bones/date.py

```diff
@@ -166,15 +164,20 @@ def singleValueFromClient(self, value: str, skel: 'viur.core.skeleton.SkeletonIn
                             value = datetime.strptime(str(rawValue), "%d.%m.%Y")
                 except:
                     value = False  # its invalid
-            if value and not self.naive:
-                value = time_zone.localize(value)
-        if value is False:
+
+        if not value:
             return self.getEmptyValue(), [
-                ReadFromClientError(ReadFromClientErrorSeverity.Invalid, "Invalid value entered")]
+                ReadFromClientError(ReadFromClientErrorSeverity.Invalid, "Invalid value entered")
+            ]
+
```

Comment:
As already talked about: This case was missing anyway

```suggestion
        if value.tzinfo and self.naive:
            return self.getEmptyValue(), [
                ReadFromClientError(ReadFromClientErrorSeverity.Invalid, "Datetime must be naive")
            ]
```